docs/developer-docs/frontend/boilerplate-frontend.mdx

```diff
@@ -0,0 +1,470 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating a frontend using a boilerplate
+
+## Overview
+
+A community-developed project can be used to create a boilerplate template for creating frontend apps using JavaScript, React, Svelte, and Vue. This guide will walkthrough how to use that boilerplate template. 
+
+## Prerequisites
+
+Before you start your project, verify the following:
+
+- [x] Download and install the IC SDK package as described in the [installing the IC SDK](/docs/current/developer-docs/setup/install) page.
+
+- [x] Download and install [git](https://git-scm.com/downloads).
+
+- [x] Download and install [Node.js](https://nodejs.org/en) <v0.12.0.
+
+### Step 1: Clone the project repository.
+
+<Tabs>
```

Comment:
nice!

Reply:
I think we should remove the tabs for the code snippets that have no change between languages though

Reply:
I think that may cause confusion for less-experienced devs who select one framework at the beginning and then the option to select disappears, as it might not be clear to them that the command will be the same for each one. I think there is a way to configure the tabs where if you select one, all the tabs on the page switch to the one you selected, which might be a good solution IMO.